docs/features/ethernet-lan.md

```diff
@@ -39,5 +40,22 @@ ESP32Deux |  | 17, 18, 23
 [WESP32](https://wesp32.com/) |  | 0, 16, 17
 [WT32-ETH01](https://www.seeedstudio.com/Ethernet-module-based-on-ESP32-series-WT32-ETH01-p-4736.html) |  | 0, 16, 18, 23
 
+## Defining GPIO for a custom PCBA
 
+The following PHY are supported by WLED according to the header ETH.h:
+[LAN8720](https://www.microchip.com/en-us/product/lan8720a)
+[TLK110](https://www.ti.com/product/TLK110)
+[IP101](https://www.lcsc.com/datasheet/lcsc_datasheet_2206021645_WIZNET-IP101GRI_C910373.pdf)
 
+The source code also mentions ETH_PHY_MAX but this is not enough information to link to a specific part
+
+If you want to add an additional board type including custom GPIO allocations to the UI, you will need to modify the following source files before compiling WLED using default_envs = esp32_eth in platformio.ini:
+| File | Line (approx) | Description |
+|---|---|---|
+const.h | 288 | increment WLED_NUM_ETH_TYPES
+const.h | 301 | add a #define for your board with the next available index after the list of existing boards
+network.cpp | 119 | add an aditional initialisation for your board type
+settings_wifi.htm | 221 | add your board name and enum index
+
+
+note that you will need to recompile the html, see [Changing web UI for instruction](https://github.com/Aircoookie/WLED/wiki/Add-own-functionality/a30b2f3004c9aafc0639138afbff73b4efe4a766)
```

Comment:
This is no longer necessary in 0.15 as it is done automatically if the environment is set up correctly.